main.js

```diff
@@ -1,13 +1,30 @@
+// const mobileMenu = document.querySelector('nav .mobile-menu');
+// const mobileClose = document.querySelector('nav .mobile-close');
+// const mobileNavigation = document.querySelector('.mobile-navigation');
+
+// mobileMenu.addEventListener('click', () => {
+//   mobileNavigation.classList.add('active');
+//   mobileClose.classList.add('active');
+// });
+
+// mobileClose.addEventListener('click', () => {
+//   mobileNavigation.classList.remove('active');
+//   mobileClose.classList.remove('active');
+// });
+
+// function display() {
+ 
+// }
```

Comment:
- [ ] Please remove this commented piece of code, It is not best practice to commit commented code , as mentioned in the best practices _**"Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history."**_